csvData/csvOutData/README.md

```diff
@@ -1,3 +1,9 @@
 # Purpose
 
 Output from data frame to be used as a model.
+
+## Contents of this Directory 
+
+* OUTPUT_Boylan_Tidepool_1.csv - the final dataframe of kate's combined data used for this project
```

Comment:
Maybe we should make it known that the combined data is from Tidepool and Medtronic - maybe say "the final dataframe of Kate's combined Tidepool & Medtronic data used for predictions" instead?

Reply:
okay that sounds good